packages/prosemirror-lwdita-backend/README.md

```diff
@@ -0,0 +1,27 @@
+# prosemirror-lwdita-backend
+API module for prosemirror lwdita
+
+This is a standalone module with it's own http server.
```

Comment:
@marmoure Can you please re-check this entire paragraph. It contains some typos.

Reply:
fixed in fe0ad59982f00b312827a75cb369cd835786c518